### ISSUE TITLE
Docs Confusion Around Universal Forwarder

### ISSUE THREAD

**Issue by sghaskell** [splunk/docker-splunk#58]
Is the Dockerhub [splunk/universalforwarder](https://hub.docker.com/r/splunk/universalforwarder/) image the same underlying image in this repo? The docs mention [valid Universal Forwarder environment variables](https://github.com/splunk/docker-splunk/blob/master/documentation/ADVANCED.md#valid-universal-forwarder-environment-variables) but `SPLUNK_ROLE=splunk_universal_forwarder` is not valid. 

<img width="972" alt="screen shot 2018-11-14 at 11 15 39 am" src="https://user-images.githubusercontent.com/8450537/48506619-b473d680-e7fe-11e8-89b9-46466e3453fc.png">

There are no links from Dockerhub to the Advanced README for `splunk/universalforwarder`. If these environment variables are valid to use with `splunk/universalforwarder`, would it be possible to update Dockerhub to point to the Advanced README?

See this section from `splunk/splunk` that links out to the Advanced README. Something like this for `splunk/universalforwarder` would be helpful.

<img width="813" alt="screen shot 2018-11-14 at 10 46 25 am" src="https://user-images.githubusercontent.com/8450537/48506211-c86b0880-e7fd-11e8-8847-a8736ef09a03.png">

If they aren't applicable, the Advanced README is misleading since the only valid forwarder role is `splunk_heavy_forwarder` and there's only one mention of **Universal Forwarder** in the entire README.

**Comment by nwang92** [comment]
Sorry for the delay in getting back to this issue - we've recently made a big push to clean up and update the documentation to be more detailed and explicit.

When using the `splunk/universalforwarder` image, the default `SPLUNK_ROLE` is set in the image to be `splunk_universal_forwarder`. So ideally, you don't need to explicitly set this parameter during use - in fact, changing it will likely have negative consequences as the binary is different and probably can't assume any other responsibilities.

I've added a note re: `splunk/universalforwarder/` and `SPLUNK_ROLE` here: https://github.com/splunk/docker-splunk/pull/137. I'll take a look at getting the Dockerhub page updated with links after we revision the docs a bit more.
